src/components/autosuggest/_autosuggest.scss

```diff
@@ -96,12 +96,12 @@
         border: 0;
         margin: 0 !important;
         .ons-panel__icon {
-            font-size: 21px;
-            line-height: 25px;
+            font-size: 1.3125rem;
+            line-height: 1.5625rem;
             margin-top: 0;
-            min-height: 24px;
-            min-width: 26px;
-            top: 17px;
+            min-height: 1.5rem;
+            min-width: 1.625rem;
+            top: 1.0625rem;
```

Comment:
font-size: 1.3rem;
line-height: 1.5rem;
margin-top: 0;
min-height: 1.5rem;
min-width: 1.5rem;
top: 1rem

Think these should be this. I dont think theres a need for them to be that specific and the `min-height` and `min-width` need to be the same value to make the icon circular